packages/twenty-server/src/modules/workflow/workflow-executor/interfaces/workflow-action.interface.ts

```diff
@@ -1,12 +1,5 @@
 import { WorkflowActionResult } from 'src/modules/workflow/workflow-executor/types/workflow-action-result.type';
-import { WorkflowStep } from 'src/modules/workflow/workflow-executor/types/workflow-action.type';
 
 export interface WorkflowAction {
-  execute({
-    step,
-    payload,
-  }: {
-    step: WorkflowStep;
-    payload?: object;
-  }): Promise<WorkflowActionResult>;
+  execute(payload: unknown): Promise<WorkflowActionResult>;
```

Comment:
- we know that payload is at least an object isn't it?

Reply:
We cannot be sure. The step could have an input `null`. Or the inferred variables could provide anything